src/hotspot/cpu/x86/vtableStubs_x86_64.cpp

```diff
@@ -217,8 +217,7 @@ VtableStub* VtableStubs::create_itable_stub(int itable_index) {
   // We expect we need index_dependent_slop extra bytes. Reason:
   // The emitted code in lookup_interface_method changes when itable_index exceeds 15.
   // For linux, a very narrow estimate would be 112, but Solaris requires some more space (130).
-  // Lilliput: expanded since load_klass is significantly and decode_klass somewhat larger.
-  const ptrdiff_t estimate = 155;
+  const ptrdiff_t estimate = 137;
```

Comment:
Upstream has it as `136`. Do we need `137`?

Reply:
I couldn't build/run it with 136, it asked for 137. *shrugs*